resources/views/auth/register.blade.php

```diff
@@ -1,74 +1,151 @@
 @extends('layouts.app')
 
-@section('content')
-<div class="container">
-    <div class="row justify-content-center">
-        <div class="col-md-8">
-            <div class="card">
-                <div class="card-header">{{ __('Register') }}</div>
+@section('title','Register')
 
-                <div class="card-body">
-                    <form method="POST" action="{{ route('register') }}">
-                        @csrf
+@section('styles')
+<link rel="stylesheet" href="{{ mix('/css/register.css') }}">
+<style> 
```

Comment:
Please transfer to your scss file

Reply:
I just sent you the scss file.please comfirm.

Reply:
P.S Imade a mistake with the branch name. This pull request is definitely the scss file for the register page.Thank you in advance.